ush/subseasonal/create_METplus_subseasonal_output_dirs.py

```diff
@@ -120,19 +120,22 @@
                                            'plot_output')
     working_dir_list.append(working_output_base_dir)
     working_dir_list.append(
-        os.path.join(working_output_base_dir,
-                     RUN+'.'+end_date_dt.strftime('%Y%m%d'))
+        os.path.join(working_output_base_dir, 'job_work_dir')
     )
     working_dir_list.append(
         os.path.join(working_output_base_dir,
-                     RUN+'.'+end_date_dt.strftime('%Y%m%d'),
-                     'logs')
+                     RUN+'.'+end_date_dt.strftime('%Y%m%d'))
     )
+    #working_dir_list.append(
```

Comment:
Can remove this section since it is commented out.

Reply:
Done!